### PR TITLE
Update logger defaults (#2457)

### PR DESCRIPTION
**Pull request by pq**
Update logger defaults 

image

Fixes: #2457

/cc @devoncarew @quangson91

### CODE REVIEW

File: src/io/flutter/logging/FlutterLogFilterPanel.java [hunk 1]
```diff
@@ -113,6 +112,7 @@ protected void customizeCellRenderer(@NotNull JList<? extends FlutterLog.Level>
                                            boolean selected,
                                            boolean hasFocus) {
         append(value.toDisplayString());
+    logLevelComboBox.setSelectedItem(FlutterLog.Level.CONFIG);
```
Comment (devoncarew) [inline src/io/flutter/logging/FlutterLogFilterPanel.java]: Needs formatting? Or, is this meant to live a few lines up?

Comment (pq) [reply src/io/flutter/logging/FlutterLogFilterPanel.java]: Oh, yeah.  That was dumb.

And actually, this should really be properly set in the prefs -- I didn't see the default there until just now.

Will fix.

Thanks!

**Comment by pq** [comment]
Fixed now.  Thanks!

**Comment by quangson91** [comment]
@pq, I have a question. Why don't we enable show color by default?
(For me, if the log levels like `warning/shout` => we should be coloring them. It is better for eyes to recognize).

**Pull request merged**